content/posts/jellyfin-10.5.md

```diff
@@ -0,0 +1,109 @@
++++
+title = "Jellyfin 10.5"
+subtitle = "A step closer to the best experience with your media"
+author = "Fernando"
+githubusername = "ferferga"
+date = "2020-03-06"
+justify = "center"
++++
+
+What would you think if I tell you that 10.5 is probably the biggest release we've done? Fasten your seat belts and get ready because the numbers are big:
+**Over 200 contributions from the community were merged and over 500 issues were fixed** since 10.4.3 (the latest stable version), both in Jellyfin and Jellyfin Web,
+the interface you access using your browser.
+
+Although we tried hard, we are even unsure if we got covered everything that's new. We hope so. Anyway, you're advised to take some popcorn before continuing. Let's dive in!
+
+We also want to apologise for the late release: We wanted to make 10.5 possible this Christmas, but we always found that something wasn't complete or finished yet.
+We hope you will forgive us (send everyone who contributed a cake if you do, please!) after trying it!
```

Comment:
As I pointed out in another comment, this informal style should not contain parentheses. But since its meant to be a joke and there is no good way to do it in a separate sentence, in this case I'd suggest "We hope you will forgive us after trying it (and maybe send us some cakes)!"